content/pages/en/privacy-policy.mdx

```diff
@@ -5,7 +5,7 @@ html_description: This privacy policy explains how, why and under what condition
 
 <p className="text__smallcaps mb__3XL">Last updated: September 26, 2023</p>
```

Comment:
So, should we update the “last updated” date for such a minor change or not? 😉

Reply:
Yep we should

Reply:
"Last update" date is not related to the importance of the changes for me.